Week of Sep29 (#411)
xregistry/spec: core/model.md

### `groups.<STRING>.resources.<STRING>.hasdocument`
- Type: Boolean (`true` or `false`, case-sensitive).
- OPTIONAL.

> - Indicates whether or not Resources of this type have a domains-specific

Author: done and did your "maybe" too

  document associated with it. If `false` then the xRegistry metadata becomes
  "the document". Meaning, a query to the Resource's URL will return the

>   xRegistry metadata and not the domain-specific document.

Author: done

  A value of `true` does not mean that these Resources are guaranteed to
  have a non-empty document, and a query to the Resource MAY return an
  empty document.

  See
  [Document Resources vs Metadata-Only Resources](./spec.md#document-resources-vs-metadata-only-resources)
  for more information.

- When not specified, the default value MUST be `true`.

> - A value of `true` indicates that Resources of this type MUST a separate
>   document to be associated with it, even if it's empty.

Reviewer: ```suggestion
- A value of `true` indicates that each Resource of this type MUST have a separate
  document associated with it, even if it's empty.
```

(At the very least, some word after "MUST", but that on its own wasn't quite enough. Happy for others to wordsmith now I've drawn attention to it.)

Author: yup, the missing "have" was the real issue, but I made the other edits too

When the directives are used in a request to update the model, the server MUST
resolve all includes prior to updating the model. The original (source)
model definition, with any "include" directives, MUST be available via
the `modelsource` attribute/entity. The expanded model (after the resolution
of any includes, and after all specification-defined attributes have been
added), MUST be available via the `model` attribute/entity. The directives MUST
only be processed during the initial update of the model. In order to have
them re-evaluated, a subsequent model update request (with those directive)

> MUST be done.

Reviewer: ```suggestion
MUST be sent.
```

(Or "performed", "executed" or something similar - but "done" doesn't feel right.)

Author: done
